doc/en/components/inputs/date-time-input.md

```diff
@@ -0,0 +1,255 @@
+
+<code-view style="height: 150px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/date-time-input-overview"
+           alt="$Platform$ Date Time Input Overview Example"
+           github-src="inputs/date-time-input/overview">
+</code-view>
+
+## Usage
+
+<!-- WebComponents -->
+First, you need to install the $ProductName$ by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+<!-- end: WebComponents -->
+
+Before using the `DateTimeInput`, you need to register it as follows:
+
+
+```ts
+import { defineComponents, IgcDateTimeInput } from 'igniteui-webcomponents';
+
+defineComponents(IgcDateTimeInput);
+```
+
+### Value binding
+The easiest way to set the value of the `IgcDateTimeInputComponent` component is by passing a Date object to the `value` property:
+
+```typescript
+const input = document.querySelector('igc-date-time-input') as IgcDateTimeInputComponent;
+const date = new Date();
+
+input.value = date;
+```
+
+The `IgcDateTimeInputComponent` also accepts [`ISO 8601`](https://tc39.es/ecma262/#sec-date-time-string-format) strings.
+
+The string can be a full `ISO` string, in the format `YYYY-MM-DDTHH:mm:ss.sssZ` or it could be separated into date-only and time-only portions.
+
+##### Date-only
+If a date-only string is bound to the `value` property of the component, it needs to be in the format `YYYY-MM-DD`. The `inputFormat` is still used when typing values in the input and it does not have to be in the same format. Additionally, when binding a date-only string, the directive will prevent time shifts by coercing the time to be `T00:00:00`.
+
+##### Time-only
+Time-only strings are normally not defined in the `ECMA` specification, however to allow the directive to be integrated in scenarios which require time-only solutions, it supports the 24 hour format - `HH:mm:ss`. The 12 hour format is not supported.
+
+##### Full ISO string
+If a full `ISO` string is bound, the directive will parse it only if all elements required by [`Date.parse`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#date_time_string_format) are provided.
+
+All falsy values, including `InvalidDate` will be parsed as `null`. Incomplete date-only, time-only, or full `ISO` strings will be parsed as `InvalidDate`.
+
+### Keyboard Navigation
+
+The `IgcDateTimeInputComponent` has intuitive keyboard navigation that makes it easy to increment, decrement, or jump through different `DateParts` among others without having to touch the mouse.
+
+|Keys|Description|
+|----|-----------|
+| <kbd>&larr;</kbd> | Move one character to the left |
+| <kbd>&rarr;</kbd> | Move one character to the right |
```

Comment:
We should probably use "to the beginning" and "to the end" instead of left and right because of the rtl support